docs/usage.md

```diff
@@ -155,8 +155,14 @@ are also required:
 -DPKEY=$PKEY
 -DIA2_ENABLE=1
 -include /path/to/generated_output_header.h
+-I $IA2_PATH/runtime/libia2/include
+-I $IA2_PATH/runtime/partition-alloc/include
 -Werror=incompatible-pointer-types
 -Wl,--wrap=pthread_create
+-Wl,--wrap=calloc
+-Wl,--wrap=malloc
+-Wl,--wrap=realloc
+-Wl,--wrap=free
```

Comment:
`runtime/partition-alloc/CMakeLists.txt` lists a few more functions from when Stephen redid our PA shim. We should add these to the README in that directory as well as here. It's not a small list so it might make sense to make a linker @ file for this.

Reply:
Also the DSOs needed these as well

Reply:
I'm going to split making an @ script into another issue (https://github.com/immunant/IA2-Phase2/issues/477), but I'll add the args in the usage docs for now 👍